flavors/template-Exasol-all-python-3.10-conda/flavor_base/udfclient_deps/packages/conda_packages

```diff
@@ -1,7 +1,6 @@
 libprotobuf|4.25.3
 zeromq|4.3.5
-openssl|3.3.0
+openssl|3.3.1
 nss|3.100
 cppzmq|4.10.0
 _openmp_mutex|4.5=2_kmp_llvm # We need this, such that we can later install libblas with mkl, which is for example needed by pytorch
```

Comment:
The question is do we still need to define this or can conda later decide which to use